src/modules/app/routes/__tests__/mainDisclaimer.spec.js

```diff
@@ -0,0 +1,13 @@
+import { goToMainDisclaimer, MAIN_DISCLAIMER_ROUTE, mainDisclaimerRoute } from '../mainDisclaimer'
+
+describe('mainDisclaimer route', () => {
```

Comment:
As a convention we usually write a name of a class or function here. What is a mainDisclaimer right now?

Reply:
Our Integreat Disclaimer, reachable with '/disclaimer'

Reply:
The naming here is because we have some endpoint names which conflict with route names, i.e. extras, categories, events.... So we would have two tests with the same name testing different stuff